cgsmiles/cgsmiles_utils.py

```diff
@@ -16,14 +16,18 @@ def find_complementary_bonding_descriptor(bonding_descriptor):
         compl = bonding_descriptor
     return compl
 
-def find_open_bonds(molecule):
+def find_open_bonds(molecule, target_nodes=None):
     """
     Collect all nodes which have an open bonding descriptor and store
     them as keys with a list of nodes as values.
     """
+    if target_nodes is None:
+        target_nodes = list(molecule.nodes)
```

Comment:
```suggestion
        target_nodes = molecule
```
You only use it to check presence of items anyway